### PR TITLE
Popover: Rename Flyout to Popover + Codemod

### PR DESCRIPTION
**Pull request by AlbertCarreras**

### CODE REVIEW

File: docs/src/Popover.doc.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ const card = (c) => cards.push(c);
 
 card(
   <PageHeader
-    name="Flyout"
-    description="Flyouts are similar to modals. They’re an alternative when there is less content to display,
+    name="Popover"
+    description="Popovers are similar to modals. They’re an alternative when there is less content to display,
```
Comment (dangerismycat) [inline docs/src/Popover.doc.js:14]: We should probably add a temporary note here that the component was formerly known as Flyout, something like "Popover (formerly known as Flyouts) are similar to modals."

File: packages/gestalt-codemods/20.0.0/__testfixtures__/flyoutAlias-rename-popover.output.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// @flow strict
+import React from 'react';
+import { Box, Popover } from 'gestalt';
```
Comment (dangerismycat) [inline packages/gestalt-codemods/20.0.0/__testfixtures__/flyoutAlias-rename-popover.output.js:3]: I don't think we want to change whatever renaming people have been doing. I think this codemod should only update the import, but not alter the renaming aspect

Comment (AlbertCarreras) [reply packages/gestalt-codemods/20.0.0/__testfixtures__/flyoutAlias-rename-popover.output.js:3]: I agree. That's how all renaming codemods we've been doing work though. Row/Stack, ScrollableContainer, GestaltProvider, etc.  I tried trying to keep the alias but I couldn't figure out. I'd try again. If it's too cumbersome, I think we just need to grep for `Flyout as` and see the use cases.

File: docs/src/Popover.doc.js (hunk 1 above)
Comment (AlbertCarreras) [reply docs/src/Popover.doc.js:14]: Got it.

File: packages/gestalt-codemods/20.0.0/__testfixtures__/flyoutAlias-rename-popover.output.js (hunk 2 above)
Comment (AlbertCarreras) [reply packages/gestalt-codemods/20.0.0/__testfixtures__/flyoutAlias-rename-popover.output.js]: DONE, I kept aliases.

File: docs/src/Popover.doc.js (hunk 1 above)
Comment (AlbertCarreras) [reply docs/src/Popover.doc.js]: DONE

**Review by dangerismycat — APPROVED** [review]
Nice work figuring out the alias issue!

**Pull request merged**